src/user/models/user/user.service.ts

```diff
@@ -64,6 +65,29 @@ export class UserService {
     return this.userRepo.findOne({ where: { address }, relations: needsRelation ? ['userData', 'wallet'] : [] });
   }
 
+  async getLinkedUser(id: number, address: string, blockchain: string): Promise<User> {
+    return this.userRepo
+      .createQueryBuilder('user')
+      .select('linkedUser.*')
+      .leftJoin('user.userData', 'userData')
+      .leftJoin('userData.users', 'linkedUser')
+      .where('user.id = :id', { id })
+      .andWhere('linkedUser.address = :address AND linkedUser.blockchain = :blockchain', { address, blockchain })
+      .getRawOne<User>();
```

Comment:
`.getOne()`?

Reply:
`.getOne()` doesn't return a user, whereby `.getRawOne<User>()` does.
Is this impacted by `SELECT linkedUser.*`?